flash/vision/segmentation/data.py

```diff
@@ -44,19 +49,38 @@
     plt = None
 
 
+def create_random_labels_map(num_classes: int) -> Dict[int, Tuple[int, int, int]]:
```

Comment:
to avoid duplication you can use it from `SegmentationLabels` (where I think it should belong) or then remove it from there and import this one